src/applications/check-in/utils/formatters/index.js

```diff
@@ -52,15 +52,18 @@ const removeTimezoneOffset = str => {
 /**
  * @param {string} items
  * @param {string} conjuction
+ * @param {boolean} addPeriod
  * @returns {string}
  */
 
-const formatList = (items, conjuction) => {
+const formatList = (items, conjuction, addPeriod = true) => {
```

Comment:
I would also accept removing the period from the function and putting those in manually where needed.

Reply:
this seemed like less impact since it only modified my usage. But yeah I can see that, we can refine this as we go.